### DISCUSSION TITLE
Si5332 Clock builder project

### DISCUSSION THREAD

**Discussion by Garripa** [ArrowElectronics/Agilex-5#1] [General]
Hi, 
Would it be possible to share the clock builder project for Si5332 pll for the AXE5-Eagle Arrow dev kit ?
We need to change the default clocks of Si5332 for our testing, and having the default clock-builder project would be a great help

**Comment by skravats** [comment]
Apologies for the delay in replying. The project is located at the link listed below.

https://github.com/ArrowElectronics/Agilex-5/blob/main/images/promo3/Si5332-GM3-RevD-AXE5-Eagle_Nov28.slabtimeproj
